src/components/brain/FilterSelect.tsx

```diff
@@ -0,0 +1,34 @@
+import { GroupBase, Select } from "chakra-react-select"
+
+export type FilterOption = {
+  value: string
+  label: string
+}
+
+type Props = {
+  FILTER_OPTIONS: Array<FilterOption>
```

Comment:
no need to pass it as prop this way, it's render independent. Just export it from the file it's defined in and import where needed. Should move the definition here and import in guildverse, it belongs here